tests/playwright/sanity/elements-panel.test.js

```diff
@@ -0,0 +1,54 @@
+const { test, expect } = require( '@playwright/test' );
+const WpAdminPage = require( '../pages/wp-admin-page.js' );
+
+test( 'add widgets from the panel by click', async ( { page }, testInfo ) => {
+	// Arrange.
+	const wpAdmin = new WpAdminPage( page, testInfo );
+	const editor = await wpAdmin.useElementorCleanPost();
+
+	// Act.
+	const heading = await addWidgetByClick( editor, 'heading' );
```

Comment:
```suggestion
	await addWidgetByClick( editor, 'heading' );
	await addWidgetByClick( editor, 'button' );
	await addWidgetByClick( editor, 'icon' );

```

Reply:
The suggestion above isn't equivalent to the original test, because he order matters in my case:

1. Add heading
2. Add button
3. Focus on heading
4. Add icon

expected: icon should be appended after the heading

Reply:
`await expect( editor.getPreviewFrame().locator( '[data-element_type="widget"]' ).nth( widget ) ).toHaveAttribute( 'data-widget_type', `${ title }.default` );}`
This will check the order. I'm not sure what is the point to focus on the heading, you will lost the focus when icon widget is added.